metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/0.2.1BETA/package/scripts/params/params_linux.py

```diff
@@ -119,16 +119,21 @@
     mysql_configname = '/etc/my.cnf'
 
 daemon_name = status_params.daemon_name
+
+install_mysql = config['configurations']['metron-env']['install_mysql']
+mysql_admin_password = config['configurations']['metron-env']['mysql_admin_password']
+
 # There will always be exactly one mysql_host
 mysql_host = config['clusterHostInfo']['metron_enrichment_mysql_server_hosts'][0]
+
 mysql_port = config['configurations']['metron-env']['metron_enrichment_db_port']
 
 mysql_adduser_path = tmp_dir + "/addMysqlUser.sh"
 mysql_deluser_path = tmp_dir + "/removeMysqlUser.sh"
 mysql_create_geoip_path = tmp_dir + "/createMysqlGeoIp.sh"
 
-enrichment_hosts = default("/clusterHostInfo/enrichment_host", [])
-enrichment_host = enrichment_hosts[0] if len(enrichment_hosts) > 0 else None
+#enrichment_hosts = default("/clusterHostInfo/enrichment_host", [])
```

Comment:
Doesn't seem like there's a reason to keep these comments around.  Can we just delete them?